src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions.test.ts

```diff
@@ -1311,8 +1311,14 @@ describe('migration actions', () => {
         maxResponseSizeBytes: 500, // set a small size to force the error
       });
       const rightResponse = (await readWithPitTask()) as Either.Right<ReadWithPit>;
-
-      await expect(Either.isRight(rightResponse)).toBe(true);
+      if (Either.isRight(rightResponse)) {
+        await expect(Either.isRight(rightResponse)).toBe(true);
+      } else {
+        console.log(
+          'got a left: ',
+          (rightResponse as Either.Left<EsResponseTooLargeError>).left.type
+        );
+      }
```

Comment:
We should also fail the test. Perhaps this logic could be:
```ts
      const rightResponse = await readWithPitTask()
      if (Either.isLeft(rightResponse)) {
        fail(`Expected a successful response but got ${rightResponse.left}`);
      }
```

Reply:
yeah this was meant as a throw-away debugging PR but CI won't run when there's console.log's and I think it'd be better for the future to have your suggestion, should be enough to find the cause.

Reply:
Also, it looks like my suggestion was wrong, the `fail` function does not actually exist in runtime (despite TS thinking it does). Looks it was removed in jest 27. Should probably just use `throw new Error(...)`!